catalyst/dl/scripts/swa.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+from argparse import ArgumentParser
+from pathlib import Path
+
+import torch
+
+from catalyst.dl.utils.swa import generate_averaged_weights
+
+
+def build_args(parser: ArgumentParser):
+    """Builds the command line parameters."""
+    parser.add_argument("--logdir", type=Path, help="Path to models logdir")
+    parser.add_argument(
+        "--models_mask", "-m", type=str, help="Pattern for models to average"
+    )
+    parser.add_argument(
+        "--save_path", type=Path, help="Path to save averaged model"
+    )
+
+    return parser
+
+
+def parse_args():
+    """Parses the command line arguments for the main method."""
+    parser = argparse.ArgumentParser()
+    build_args(parser)
+    args = parser.parse_args()
+    return args
+
+
+def main(args, _):
+    """Main method for ``catalyst-dl swa``."""
+    logdir: Path = args.logdir
+    models_mask: str = args.models_mask
+    save_path: Path = args.save_path
+
+    averaged_weights = generate_averaged_weights(
+        logdir, models_mask, save_path
+    )
+
+    torch.save(averaged_weights, str(save_path / "swa_weights.pth"))
+
+
+if __name__ == "__main__":
+    main(parse_args(), None)
```

Comment:
could you please also add bash tests here github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_cv.sh#L41L44

something like
```bash
PYTHONPATH=./examples:./catalyst:${PYTHONPATH} \
  python catalyst/dl/scripts/swa.py \
  --logdir=${LOGDIR} --output-path=./swa.pth
```
?